scripts/pipelines/build.toolchain-feeds.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```

Comment:
Should the name of the script be just `build.toolchain`? I don't think we need the `feeds`.